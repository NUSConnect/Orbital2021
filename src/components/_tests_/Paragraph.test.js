import * as React from 'react'
import Paragraph from '../Paragraph'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<Paragraph />)
  expect(tree).toMatchSnapshot()
})
