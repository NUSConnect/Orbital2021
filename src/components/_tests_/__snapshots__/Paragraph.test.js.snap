// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000000",
        "fontFamily": "System",
        "fontWeight": "400",
      },
      Object {
        "textAlign": "left",
      },
      Object {
        "fontSize": 15,
        "lineHeight": 21,
        "marginBottom": 12,
        "textAlign": "center",
      },
    ]
  }
/>
`;
